deploy/kubernetes/base/daemonset.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1beta2 # for versions before 1.8.0 use extensions/v1beta1 [apps/v1beta2]
+kind: DaemonSet
+metadata:
+  name: cadvisor
+spec:
+  template:
+    spec:
+      containers:
+      - name: cadvisor
```

Comment:
I'm surprised cAdvisor doesn't need to run as privileged. I guess that's just because it reads everything from the mounted hostPath volumes?

Reply:
yeah, you should know better than me, but I thought privileged only mattered for making syscalls.  cAdvisor just reads lots of stuff in sys/fs/cgroup